examples/src/components/navigation-menu/navigation-menu.tsx

```diff
@@ -31,7 +31,7 @@ export const navigationMenuClassNames = {
 const useStyles = makeStyles({
   root: {
     height: "100%",
-    width: "300px",
+    width: "200px",
```

Comment:
needed?

Reply:
No. It helped me test the slider with our use cases and I saw no reason why it would need to be wider. I can revert it.